app/app.go

```diff
@@ -317,12 +320,14 @@ func NewEthermintApp(
 	// init params keeper and subspaces
 	app.ParamsKeeper = initParamsKeeper(appCodec, cdc, keys[paramstypes.StoreKey], tkeys[paramstypes.TStoreKey])
 	// set the BaseApp's parameter store
-	bApp.SetParamStore(app.ParamsKeeper.Subspace(baseapp.Paramspace).WithKeyTable(paramstypes.ConsensusParamsKeyTable()))
+	authAddr := authtypes.NewModuleAddress(govtypes.ModuleName).String()
```

Comment:
this is effectively setting up permissions for the gov module to be able to modify consensus params? might be worth a comment for future readers